pages/index.html

```diff
@@ -291,6 +302,7 @@ <h2 class="bannerbox__title">I can offer a ride</h2>
                         </label>
                         <small>I understand that Carpool Vote LCC will share my contact details with the driver if there's a match. (Carpool Vote LLC will not share personal details with anybody else, unless required by law, and will destroy them within three months of election day of you've asked us not to stay in touch.)</small>
                         <small>I understand that Carpool Vote LLC provides introductions between riders and volunteer drivers who have signed up on the platform. I understand that anybody can sign up to drive and Carpool Vote LLC is unable to perform any background checks on people who use the platform. As with any other environment where I meet new people, I will take steps to keep myself and my possessions safe and accept that Carpool Vote LLC cannot be responsible if anything goes wrong.</small>
+                        <small>I understand that Carpool Vote cannot guarantee that I will find appropriate matches through the platform, or that any agreed ride will occur. I will take any actions</small>
```

Comment:
Looks like the text here doesn't match the issue. Can you please update to the following?
> I understand that Carpool Vote cannot guarantee that I will find appropriate matches through the platform, or that any agreed ride will occur. I take full responsibility for any cost related to using the platform.